coraza.conf-recommended

```diff
@@ -45,7 +45,7 @@ SecRequestBodyLimit 13107200
 
 SecRequestBodyInMemoryLimit 131072
 
-SecRequestBodyNoFilesLimit 131072
+# SecRequestBodyNoFilesLimit 131072
```

Comment:
Being not implemented, I think we should at least comment it out, not letting users think that it is an enforced configuration

Reply:
Let's add a comment too, `# Coraza currently does not read this parameter` I think

Reply:
Could you also open an issue for this? Are we supporting it @jptosso ?

Reply:
I added the following comment: `# SecRequestBodyNoFilesLimit is currently not supported by coraza`. As far as I can see, we actually just read the parameter, but we do not enforce any logic based on this.
Opened https://github.com/corazawaf/coraza/issues/896 about it